source/includes/steps-connect-looker.yaml

```diff
@@ -1,8 +1,8 @@
-title: Add whitelist entries to Atlas for Looker.
-ref: add-whitelist-entries
+title: Add access list entries to Atlas for Looker.
+ref: add-accesslist-entries
 stepnum: 1
 content: |
-   `Add whitelist entries <https://docs.atlas.mongodb.com/security-whitelist/index.html>`_
+   `Add access list entries <https://docs.atlas.mongodb.com/security-accesslist/index.html>`_
```

Comment:
```suggestion
   `Add access list entries <https://www.mongodb.com/docs/atlas/security/ip-access-list/>`_
```
The url you have here does not seem to exist. If you navigate to the original url it redirects to the one I suggested here.